@pedrobaeza The `write` functions are not triggered during the tests of `mass_mailing_custom_unsubscribe`, but it seems that the definition of the `active` field on `mailing.contact.subscription` is breaking this test: 
```
AssertionError: The test code "odoo.__DEBUG__.services['web_tour.tour'].run('mass_mailing_custom_unsubscribe_tour_contact')" failed
Tour mass_mailing_custom_unsubscribe_tour_contact failed at step Uncheck list 1 (trigger: li:contains('test list 1') input:checked)
```
Question is, can these JS tests be fixed considering the possibility of having an  `active` field on `mailing.contact.subscription` or should we better test the modules separately :question:

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -36,18 +36,18 @@ jobs:
       matrix:
         include:
           - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
-            include: "mail_restrict_follower_selection"
+            include: "mail_restrict_follower_selection,mass_mailing_contact_active,mass_mailing_partner_contact_active"
             makepot: "true"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
-            include: "mail_restrict_follower_selection"
+            include: "mail_restrict_follower_selection,mass_mailing_contact_active,mass_mailing_partner_contact_active"
             name: test with OCB
           - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
-            exclude: "mail_restrict_follower_selection"
+            exclude: "mail_restrict_follower_selection,mass_mailing_contact_active,mass_mailing_partner_contact_active"
             makepot: "true"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
-            exclude: "mail_restrict_follower_selection"
+            exclude: "mail_restrict_follower_selection,mass_mailing_contact_active,mass_mailing_partner_contact_active"
             name: test with OCB
     services:
       postgres:

